pkg/controller/map/storage/validation.go

```diff
@@ -138,23 +137,10 @@ func (r *Reconciler) validateDestination(mp *api.StorageMap) (err error) {
 		return
 	}
 	notValid := []string{}
-	var resource interface{}
-	switch provider.Type() {
-	case api.OpenShift:
-		resource = &ocp.StorageClass{}
-	case api.VSphere:
-		return
-	default:
-		err = liberr.Wrap(
-			web.ProviderNotSupportedError{
-				Provider: provider,
-			})
-		return
-	}
```

Comment:
Resource no longer needed because using inventory.Network() instead of inventory.Get().